<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>861</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>260</width>
       <height>150</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid black;
font: 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>photo/New Project.png</normaloff>photo/New Project.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>260</width>
       <height>150</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid black;
font: 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>photo/settings.png</normaloff>photo/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>260</width>
       <height>150</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid black;
font: 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>photo/history-512.png</normaloff>photo/history-512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>851</width>
       <height>751</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:none none none none;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>71</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string>  URL</string>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="urlText">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>711</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="download">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>791</width>
         <height>100</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string>DOWNLOAD</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>791</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
     border: 2px solid black;
     border-radius: 4px;
 
 }

 QProgressBar::chunk {
     background-color: rgb(210, 252, 0);
	 border-radius:4px;
     width: 20px;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QLabel" name="pathLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string>PATH</string>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="pathField">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>711</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="path_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>791</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;
border-radius:4px;
color: rgb(210, 252, 0);</string>
       </property>
       <property name="text">
        <string>SET PATH</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>831</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QHeaderView::section {

border:2px solid black;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:4px;
color: rgb(210, 252, 0);
background-color: rgb(35, 35, 35);
}

QTableCornerButton::section {
background-color: rgb(35, 35, 35);
}

QTableView::item {

color: rgb(210, 252, 0);
qproperty-defaultAlignment: AlignVCenter;
background-color: black;
border:1px solid #D2FC00;
border-style:none solid solid none;
}

</string>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Link</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
